docs/source/pillars/computing_technology.md

```diff
@@ -350,8 +350,8 @@ This {term}`business process <business process>` involves forecasting and determ
   - Guidance
   - Importance
 * - 2.3.1.
-  - You must ensure that all projects understand what resources are available and what the associated costs will be before the project starts.
-  - For on-premises systems this might be related to the available hardware, for cloud-based systems there might be limits on how many instances of a particular resource (_e.g._ GPUs) can be used
+  - You must ensure that all project members understand what resources are available and what the associated costs will be before the project starts.
+  - For on-premises systems this might be related to the available hardware, for cloud-based systems there might be limits on how many instances of a particular resource (_e.g._ GPUs) can be used.
```

Comment:
```suggestion
  - For on-premises systems this might be related to the available hardware. For cloud-based systems there might be limits on how many instances of a particular resource (_e.g._ GPUs) can be used.
```